frontend/src/app/helpers/endpoints.ts

```diff
@@ -1,7 +1,11 @@
 export const USERS = "/users";
 
+export const COMS = "/api/v1/user"
+
 export const GRAPHQL= "/graphql"
 
 export const API:string = process.env.REACT_APP_BACKEND_API|| ((window as any)._env_ && (window as any)._env_.REACT_APP_BACKEND_API) 
 
-export const USERS_API: string =  process.env.REACT_APP_BACKEND_USERS_API|| ((window as any)._env_ && (window as any)._env_.REACT_APP_BACKEND_USERS_API) 
\ No newline at end of file
+export const USERS_API: string =  process.env.REACT_APP_BACKEND_USERS_API|| ((window as any)._env_ && (window as any)._env_.REACT_APP_BACKEND_USERS_API) 
+
+export const COMS_API: string =  process.env.REACT_APP_BACKEND_COMS_API|| ((window as any)._env_ && (window as any)._env_.REACT_APP_BACKEND_COMS_API) 
```

Comment:
Use REACT_APP_BCBOX_BASE_URL as we already have one key

Reply:
I have added the suggested key and push the code